packages/mds-agency/tests/request-handlers.ts

```diff
@@ -143,7 +143,7 @@ describe('Agency API request handlers', () => {
     const getFakeBody = () => {
       const device_id = uuid()
       const vehicle_id = uuid()
-      const type: Device['type'] = 'car'
+      const type: Device['type'] = 'carshare'
```

Comment:
Mind updating this to use `VEHICLE_TYPES.carshare`?

Reply:
Done